api/src/main/java/com/github/retrooper/packetevents/protocol/player/storage/StorageValueId.java

```diff
@@ -0,0 +1,26 @@
+package com.github.retrooper.packetevents.protocol.player.storage;
+
+public final class StorageValueId extends StorageIdBase {
```

Comment:
is there any reason for this to exist?

Reply:
Can be useful for example if a developer wants to store different types of data throught different parts of joining, where the automatic casting would be only an obstacle